bcipy/helpers/tests/resources/mock_session/parameters.json

```diff
@@ -46,14 +46,6 @@
     "recommended_values": "",
     "type": "int"
   },
-  "eeg_buffer_len": {
-    "value": "2.00",
```

Comment:
TODO: add prestim_length or pull from root

Reply:
We had discussed removing this `parameters.json` and just using `bcipy/parameters/parameters.json` always (so that the tests don't become stale/so we don't have to remember to update parameters in 2 places) - does this still sound good to you?

Reply:
Yes - I can do that work for this PR!

Reply:
We talked about this one and it is taken care of